button/button.html

```diff
@@ -42,6 +42,18 @@
         <button class="btn btn-hover btn-outline-warning">Hoverable Outline Warning Button</button>
         <button class="btn btn-hover btn-outline-danger">Hoverable Outline Danger Button</button>
     </div>
+
+    <div>
+        <button class="btn btn-outline-secondary"><a href="#">Link Button</a></button>
+    </div>
+
+    <div>
+        <button class="btn btn-secondary"><img src="download_white_24dp.svg" alt=""></button>
```

Comment:
Please provide alt value for accessibility.

Reply:
yes, thank you  !

Reply:
> Please provide alt value for accessibility.

DONE